ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/permit-simulation.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { NameType } from '@metamask/name-controller';
+
+import { Numeric } from '../../../../../../../../shared/modules/Numeric';
+import Name from '../../../../../../../components/app/name/name';
+import {
+  ConfirmInfoRow,
+  ConfirmInfoRowText,
+} from '../../../../../../../components/app/confirm/info/row';
+import { useI18nContext } from '../../../../../../../hooks/useI18nContext';
+import { currentConfirmationSelector } from '../../../../../../../selectors';
+import { Box, Text } from '../../../../../../../components/component-library';
+import {
+  BackgroundColor,
+  BorderRadius,
+  Display,
+  TextAlign,
+} from '../../../../../../../helpers/constants/design-system';
+import { SignatureRequestType } from '../../../../../types/confirm';
+import useTokenExchangeRate from '../../../../../../../components/app/currency-input/hooks/useTokenExchangeRate';
+import { IndividualFiatDisplay } from '../../../../simulation-details/fiat-display';
+
+const PermitSimulation: React.FC = () => {
+  const t = useI18nContext();
+  const currentConfirmation = useSelector(
+    currentConfirmationSelector,
+  ) as SignatureRequestType;
+  const [faitValue, setFiatValue] = useState<number>();
```

Comment:
```suggestion
  const [fiatValue, setFiatValue] = useState<number>();
```
`faitValue` → `fiatValue` and below